src/util/misc.rs

```diff
@@ -10,6 +10,52 @@ use std::time::{SystemTime, UNIX_EPOCH};
 use crate::args::Args;
 use super::db::delete;
 
+pub fn cleanup_orphaned_files(pastas: &Vec<Pasta>, args: &Args) {
+    log::debug!("Starting orphaned files cleanup");
+    
+    let attachments_dir = format!("{}/attachments", args.data_dir);
+    let dirs = match std::fs::read_dir(&attachments_dir) {
+        Ok(dirs) => dirs,
+        Err(e) => {
+            log::debug!("No attachments directory found or unable to read: {}", e);
```

Comment:
Ideally orphaned files should return an error which gets logged by the caller rather then logging in orphaned files on error. 

I'm okay with merging this without that change though. If you do want to make that change do read https://doc.rust-lang.org/book/ch09-00-error-handling.html before.